css/index.css

```diff
@@ -1 +1,304 @@
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+body {
+  line-height: 1;
+}
+ol,
+ul {
+  list-style: none;
+}
+blockquote,
+q {
+  quotes: none;
+}
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: '';
+  content: none;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+/* Set every element's box-sizing to border-box */
+* {
+  box-sizing: border-box;
+}
+html,
+body {
+  height: 100%;
+  font-size: 62.5%;
+  font-family: arial;
+}
+.container {
+  font-size: 1.6rem;
+  max-width: 1000px;
+  margin: 0 auto;
+}
+.container h1 {
+  font-size: 5rem;
+}
+.container p {
+  font-size: 1.6rem;
+}
+.container nav {
+  display: flex;
+  width: 35%;
+  justify-content: space-between;
+}
+.container nav a {
+  text-decoration: none;
+  color: black;
+}
+.container header {
+  display: flex;
+  flex-direction: row-reverse;
+  width: 100%;
+  margin: 20px auto;
+}
+@media (max-width: 500px) {
+  .container header {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+  .container header nav {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+  }
+  .container header nav a {
+    width: 100%;
+    color: black;
+    border: 1px solid grey;
+    padding: 10px;
+    font-size: 2rem;
+    margin: 2px auto;
+    text-align: center;
+  }
+  .container header nav a:hover {
+    background-color: grey;
+  }
+}
+.container .top {
+  width: 100%;
+  margin: auto;
+  padding-bottom: 100px;
+  background-color: #E1EAEC;
+}
+.container .top img {
+  margin-top: 50px;
+  width: 30%;
+}
+.container .top h3 {
+  font-size: 2.5rem;
+}
+.container .top h2 {
+  font-size: 2.5rem;
+}
+.container .top .topdiv {
+  border-bottom: 1px solid grey;
+}
+@media (max-width: 500px) {
+  .container .top .topdiv h3 {
+    text-align: center;
+  }
+  .container .top .topdiv h1 {
+    text-align: center;
+  }
+  .container .top .topdiv img {
+    display: flex;
+    margin: 20px auto;
+  }
+  .container .top .div2 {
+    display: flex;
+    flex-direction: column;
+  }
+  .container .top .div2 button {
+    text-align: center;
+    padding: 10px 30px;
+  }
+  .container .top .div2 p {
+    margin-bottom: 20px;
+  }
+}
+.container div {
+  width: 90%;
```

Comment:
I usually like to keep default width (100%), and then use max-width and margin: 0 auto to narrow the width of my container.